docs/features.md

```diff
@@ -274,6 +274,14 @@ Evaluates code blocks in comments with a click. [Tutorial](https://github.com/ha
 
 ![Eval Demo](https://raw.githubusercontent.com/haskell/haskell-language-server/master/plugins/hls-eval-plugin/demo.gif)
 
+Known limitations:
+
+- Standard [input](https://github.com/haskell/haskell-language-server/issues/2913)
+  and [(error) output](https://github.com/haskell/haskell-language-server/issues/1977)
```

Comment:
Error output does not break HLS communication, perhaps it would be more accurate to say that it is not included in the evaluation result.

Reply:
Correct, I had to check to convince myself, but (error) output does go to HLS output, it just isn't captured.

Fixed :heavy_check_mark: